### PR TITLE
Master merge to develop

### PR DESCRIPTION
**Pull request by gbalint**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: facf1bc9d66cbe4bb5492486a6fd876a13eeb31b
  * Your commit message body contains a line that is longer than 72 characters
  * Your subject line is longer than 72 characters

 * Commit: d483da766f812ab171ad611e4c85bf5b936b5d94
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/ethersphere/go-ethereum

---
This message was auto-generated by https://gitcop.com

**Comment by acud** [comment]
@nonsense can you explain how is it that gitcop is policing commit messages that were already merged to upstream? is the config different?

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: facf1bc9d66cbe4bb5492486a6fd876a13eeb31b
  * Your commit message body contains a line that is longer than 72 characters
  * Your subject line is longer than 72 characters

 * Commit: d483da766f812ab171ad611e4c85bf5b936b5d94
  * Your subject line is longer than 72 characters

Guidelines are available at https://github.com/ethersphere/go-ethereum

---
This message was auto-generated by https://gitcop.com

**Comment by nonsense** [comment]
@justelad GitCop is checking every single commit from the branch you want to merge to the base branch.

The fuck-up between ethersphere and go-ethereum happened on commit:
```
swarm/api/http: http package refactoring 1/5 and 2/5 (#17164)
```

I am not sure how exactly it happened, but on go-ethereum, the author is me and Balint, whereas on ethersphere you are the author. My guess is that I forgot to force-push to develop branch, while waiting for Balint to give the permissions, and you pushed to ethersphere/develop in the meantime.

-----

Bottom line:
1. We have to resolve these conflict (merging master back to develop).
2. We have to merge any outstanding changes in develop back to go-ethereum/master.
3. We have to stop using the develop branch.

I hope this makes sense.

**Comment by acud** [comment]
@nonsense makes sense and agreed. Let’s try to make a date for tomorrow afternoon to resolve the conflicts and merge to develop then back to upstream

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: facf1bc9d66cbe4bb5492486a6fd876a13eeb31b
  * Your commit message body contains a line that is longer than 72 characters
  * Your subject line is longer than 72 characters

 * Commit: d483da766f812ab171ad611e4c85bf5b936b5d94
  * Your subject line is longer than 72 characters

 * Commit: 5d30be412b0f9181cb007c8893ee583ee4319116
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/ethersphere/go-ethereum

---
This message was auto-generated by https://gitcop.com

**Comment by nonsense** [comment]
@justelad yes, let's do that. Ping me when you are around, I want to do this ASAP.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: facf1bc9d66cbe4bb5492486a6fd876a13eeb31b
  * Your commit message body contains a line that is longer than 72 characters
  * Your subject line is longer than 72 characters

 * Commit: d483da766f812ab171ad611e4c85bf5b936b5d94
  * Your subject line is longer than 72 characters

 * Commit: 5d30be412b0f9181cb007c8893ee583ee4319116
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/ethersphere/go-ethereum

---
This message was auto-generated by https://gitcop.com

**Comment by acud** [comment]
resolved by #797
